com.unity.uiwidgets/Runtime/ui/native_bindings.cs

```diff
@@ -64,4 +64,14 @@ public abstract class NativeWrapperDisposable : NativeWrapper, IDisposable {
             _dispose();
         }
     }
+
+    public abstract class NativeWrapperCPtrDisposable : NativeWrapperDisposable {
+        public bool isDisposed = false;
+
+        protected NativeWrapperCPtrDisposable() {
```

Comment:
we can also put DisposeCPtr() and DisposePtr() here.

Gnenrally, 
DisposeCPtr() == DisposePtr(_ptr)
DisposePtr() is "if .... " + base.DisposePtr()